fieldservice_route/models/fsm_route_dayroute.py

```diff
@@ -13,6 +14,13 @@ class FSMRouteDayRoute(models.Model):
     def _get_default_person(self):
         return self.route_id.fsm_person_id.id or False
 
+    @api.model
+    def _get_default_date_start_planned(self):
```

Comment:
s/date/datetime ?

Reply:
@dreispt The field is named `date_start_planned`.